README.md

```diff
@@ -1,3 +1,19 @@
+<!---
+Copyright 2023- The HuggingFace Team. All rights reserved.
```

Comment:
```suggestion
Copyright 2022 - The HuggingFace Team. All rights reserved.
```

Reply:
I think this should just be when the doc was authored, which in this case, is when Diffusers was created